lint.whitelist

```diff
@@ -905,6 +905,7 @@ MISSING-LINK: css/cssom-view/elementFromPoint-002.html
 MISSING-LINK: css/cssom-view/elementFromPoint-003.html
 MISSING-LINK: css/cssom-view/elementFromPoint.html
 MISSING-LINK: css/cssom-view/elementScroll.html
+MISSING-LINK: css/cssom-view/elementScroll-002.html
```

Comment:
Instead of adding an exception, this test should contain links to the relevant specification like https://github.com/tigercosmos/web-platform-tests/blob/3f589840950e74cb4918709cdcb6b6e59c57565a/css/cssom-view/cssom-getBoundingClientRect-002.html#L7-L8 .